shared/bundle_analysis/parser.py

```diff
@@ -87,8 +105,8 @@ def _parse_event(self, event: Tuple[str, str, str]):
             if bundle is None:
                 bundle = Bundle(name=value)
                 self.db_session.add(bundle)
```

Comment:
Fixes a bug where only save the bundle's new session if the bundle doesn't exist (ie new upload)